packages/lab/src/dialog/DialogHeader.css

```diff
@@ -0,0 +1,47 @@
+/* Styles applied to the root element */
+.saltDialogHeader {
```

Comment:
If there are multiple dialog headers mounted this will cause an issue. Should we worry?

Reply:
:S -next?

Reply:
it would change bottom padding for the old ones from 100 to 300